pkg/security/podsecuritypolicy/util/util.go

```diff
@@ -182,8 +183,9 @@ func PSPAllowsHostVolumePath(psp *extensions.PodSecurityPolicy, hostPath string)
 		return true
 	}
 
+	path := filepath.Clean(hostPath)
```

Comment:
To @liggitt's point, could we do checks and/or cleaning on the path in the kubelet? Similar to how we check for root user in the kubelet?